basics_scenarios/s3_batch/SPECIFICATION.md

```diff
@@ -0,0 +1,223 @@
+# Amazon S3 Batch Basics Scenario Specification
+
+## Overview
+
+This document outlines the technical specifications for the _Amazon S3 Batch Basic Scenario_, a scenario designed to demonstrate Amazon S3 Batch opertions functionality and SDKs. It is primarily intended for the AWS code examples team to use while developing this example in additional languages.
+
+
+## Resources
+The required resources is an IAM role that is automatically created using an AWS CloudFormation template.
```

Comment:
```suggestion
The required resource is an IAM role that is automatically created using an AWS CloudFormation template.
```

Reply:
Also, I didn't see any comments in the helper file or instructions here explaining how to pass stackName and templatePath as arguments or configurations.

Reply:
As discussed - this is automactially done.